addOns/alertFilters/src/main/resources/org/zaproxy/zap/extension/alertFilters/resources/Messages.properties

```diff
@@ -1,5 +1,57 @@
-alertFilters.api.action.addAlertFilter = Adds a new alert filter for the context with the given ID. 
-alertFilters.api.action.addGlobalAlertFilter = Adds a new global alert filter. 
+alertFileters.api.action.addAlertFilter.param.attack = The attack value for which the filter should apply (can be regex).
```

Comment:
There's a typo in the keys. Better add a test to make sure the keys are correct:
https://github.com/zaproxy/zap-extensions/blob/505c6930d37aa8b21346081c278ad1e33ba7a990/addOns/client/src/test/java/org/zaproxy/addon/client/spider/ClientSpiderApiUnitTest.java#L129-L182